iOS/Airbnb_iOS/Airbnb_iOS/Browsing/Controller/BrowseViewController.swift

```diff
@@ -55,15 +55,14 @@ extension BrowseViewController: UISearchBarDelegate {
 
     func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
         guard searchText != "" else {
-            browsingSpotDataSource.removeAllResults()
             DispatchQueue.main.async {
                 self.browsingSpotCollectionView.collectionView.reloadData()
             }
             return
         }
 
         self.changeCollectionViewToSearchingView()
-        searchCompleter.queryFragment = searchText
+        searchDataManager.getQuearyFragment(text: searchText)
```

Comment:
값을 설정하는 함수에서 get 동사가 어울리지 않네요. set이나 update 어떨까요?

```suggestion
        searchDataManager.updateQuearyFragment(searchText) or
        searchDataManager.setQuearyFragment(with: searchText)
```

Reply:
get보다는 말씀해주신 update가 더 적절한 것 같습니다.
해당 메서드명으로 수정해보겠습니다^^